lib/gems/pending/appliance_console/date_time_configuration.rb

```diff
@@ -12,10 +12,10 @@ class DateTimeConfiguration
 
     include ApplianceConsole::Logging
 
-    def initialize
-      @new_date         = nil
-      @new_time         = nil
-      @manual_time_sync = false
+    def initialize(config = {})
+      @new_date         = config.has_key(:date) ? config[:date] : nil
```

Comment:
`has_key` should be `key?` ... not sure what class this is, but Hashes have the `key?` method which is preferable.